connection.go

```diff
@@ -740,30 +725,10 @@ func (c *Connection) updateLastActivity(frame *Frame) {
 	}
 }
 
-// pendingExchangeMethodAdd returns whether the method that is trying to
-// add a message exchange can continue.
-func (c *Connection) pendingExchangeMethodAdd() bool {
-	return c.pendingMethods.Inc() > 0
-}
-
-// pendingExchangeMethodDone should be deferred by a method called
-// pendingExchangeMessageAdd.
-func (c *Connection) pendingExchangeMethodDone() {
-	c.pendingMethods.Dec()
-}
-
-// closeSendCh waits till there are no other goroutines that may try to write
+// closeStopCh waits till there are no other goroutines that may try to write
```

Comment:
This comment isn't entirely accurate anymore.

Can we also add that closing the `sendCh` is not needed and ref 93ef5c1 as well?

Reply:
Removed this method completely as it's pretty trivial now, and added comments to the only place this is used.